allwinner-hal/src/dma/register.rs

```diff
@@ -27,31 +27,353 @@ pub struct RegisterBlock {
 #[repr(C)]
 pub struct ChannelRegisterBlock {
     /// DMAC Channel Enable Register.
-    pub enable: RW<u32>,
+    pub enable: RW<ChannelEnableRegister>,
```

Comment:
```suggestion
    pub enable: RW<ChannelEnable>,
```

Remove redundant `Register` in the structure names.